### PR TITLE
RELATED: RAIL-298 Vis. library works with new AFM

### PR DESCRIPTION
**Pull request by ocetnik**
@gooddata/indigo-visualizations@0.16.0-tomasreichmann-ocetnik-doc-tre-new-afm-2017-11-27T13-33-30-381Z

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/184/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/837/ : SUCCESS in 3m 05s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/95/ : FAILURE in 3s (non-voting)

File: src/Table/utils/dataTransformation.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            type: 'attrLabel',
+            id: header.localIdentifier,
+            identifier: header.localIdentifier,
+            uri: header.uri,
+            title: header.name
+        };
+    }
+
+    return {
+        type: 'metric',
```
Comment (petrjanu) [inline src/Table/utils/dataTransformation.js]: Is 'metric' okay here?

Comment (ocetnik) [reply src/Table/utils/dataTransformation.js]: yes, we need to be backward compatible

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/188/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/839/ : FAILURE in 2m 56s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/99/ : FAILURE in 3s (non-voting)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/190/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/841/ : FAILURE in 1m 12s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/101/ : FAILURE in 3s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/191/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/842/ : SUCCESS in 1m 08s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/102/ : FAILURE in 2s (non-voting)

**Comment by ocetnik** [comment]
extended test - cafe

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/193/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/137/ : FAILURE in 12s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/192/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/843/ : FAILURE in 48s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/103/ : FAILURE in 7s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/194/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/844/ : SUCCESS in 1m 13s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/104/ : FAILURE in 2s (non-voting)

**Comment by ocetnik** [comment]
extended test - gemini

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/196/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/846/ : FAILURE in 53s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/106/ : FAILURE in 1m 15s (non-voting)

**Comment by osklive-kacatko** [comment]
Link to Gemini test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-gemini-zuul/145/artifact/gemini/reports/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/197/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/138/ : FAILURE in 4m 33s

**Comment by tomasreichmann** [comment]
extended test - gemini

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/198/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/847/ : FAILURE in 1m 48s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/107/ : FAILURE in 1m 15s (non-voting)

**Comment by osklive-kacatko** [comment]
Link to Gemini test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-gemini-zuul/146/artifact/gemini/reports/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/199/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/139/ : FAILURE in 4m 35s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/200/ : SUCCESS in 25s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/848/ : FAILURE in 51s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/108/ : FAILURE in 2m 43s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/201/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/849/ : SUCCESS in 1m 37s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/109/ : SUCCESS in 1m 46s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/202/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/850/ : SUCCESS in 1m 14s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/110/ : SUCCESS in 1m 41s (non-voting)

**Comment by lhorak** [comment]
extended test - gemini

**Comment by osklive-kacatko** [comment]
Link to Gemini test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-gemini-zuul/147/artifact/gemini/reports/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/203/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/140/ : FAILURE in 4m 38s

**Comment by tomasreichmann** [comment]
extended test - gemini

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/204/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/851/ : SUCCESS in 2m 05s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/111/ : SUCCESS in 1m 46s (non-voting)

**Comment by osklive-kacatko** [comment]
Link to Gemini test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-gemini-zuul/148/artifact/gemini/reports/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/205/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/141/ : SUCCESS in 6m 32s

**Comment by tomasreichmann** [comment]
@gooddata/indigo-visualizations@0.16.0-tomasreichmann-ocetnik-doc-tre-new-afm-2017-11-23T13-52-34-552Z

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/206/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/852/ : FAILURE in 3m 25s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/112/ : FAILURE in 3m 26s (non-voting)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/207/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/853/ : SUCCESS in 1m 16s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/113/ : SUCCESS in 1m 35s (non-voting)

**Comment by tomasreichmann** [comment]
extended test - gemini

**Comment by osklive-kacatko** [comment]
Link to Gemini test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-gemini-zuul/149/artifact/gemini/reports/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/209/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/142/ : FAILURE in 6m 31s

**Comment by tomasreichmann** [comment]
extended test - gemini

**Comment by tomasreichmann** [comment]
extended test - gemini

**Comment by tomasreichmann** [comment]
extended test - gemini

**Comment by tomasreichmann** [comment]
extended test - gemini

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/214/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/145/ : FAILURE in 1h 18m 41s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/213/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/854/ : FAILURE in 41s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/116/ : FAILURE in 2m 18s (non-voting)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/215/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-el7/855/ : FAILURE in 35s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/117/ : FAILURE in 1m 01s (non-voting)

**Pull request closed without merging**